torch/_refs/__init__.py

```diff
@@ -424,6 +425,30 @@ def _fn(*args, **kwargs):
     return _fn
 
 
+def _make_inplace(fn):
+    """
+    Given a function with out variant (i.e. using `out_wrapper()), it returns its in-place variant
+    """
+
+    # nb. We use the name of the first argument used in the unary references
```

Comment:
Style nit: boo to adding more Italian

Reply:
I literally learnt of this expression in the last years, while reading through the PyTorch codebase. You'll have to point your finger to other people if you want to find the root of the issue :P